### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by jhlegarreta**
Create an `ISSUE_TEMPLATE.md` file to provide a guide when submitting issues to ITK.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+[Which your OS and compiler are]
+
+
+### Additional Information
+
+[Any additional information, configuration or data that might be necessary to reproduce the issue.]
+
+### Label
+
+[Choose the label that best fits the issue]
```
Comment (hjmjohnson) [inline ISSUE_TEMPLATE.md]: Give a list of labels here to choose from.  These labels should match the GITHUB labels for issues.

Comment (hjmjohnson) [reply ISSUE_TEMPLATE.md]: Or at least give a URL of where to find the labels.

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+### Milestones
```
Comment (hjmjohnson) [inline ISSUE_TEMPLATE.md]: Give a URL of where to find the list of Milestones.

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jhlegarreta) [reply ISSUE_TEMPLATE.md]: @hjmjohnson thanks for the review ! This was just a proposal; the question is whether we will effectively be using labels. The VTK folks do have some, and they do use them frequently. If for any reason it is decided that we will not be using labels (despite I'd be for proposing/using them), then adding such a section is not worthwhile.

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (jhlegarreta) [reply ISSUE_TEMPLATE.md]: Same comment as for the labels.

**Comment by jhlegarreta** [comment]
An additional comment as for checking the ITK version: we could run `ctest -V -R itkVersionTest -C` but looks like an overkill, and the user may not have built the tests/may have had ITK installed (i.e. not compiled). Inspecting the main `CMakeLists.txt` may be invalid if having ITK installed.

Any thoughts or improvements?

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+[Before submitting an issue, please check that your isssue hasn't been already filed]
```
Comment (thewtex) [inline ISSUE_TEMPLATE.md]: isssue -> issue

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (thewtex) [reply ISSUE_TEMPLATE.md]: Maybe we can add the *Label* and *Milestones* section later once we have figured out what labels to use and sorted out the milestoning system.

**Comment by jhlegarreta** [comment]
Done. Commit 7970cf06da4da970da10c0bf75d255639e317e26 removes the `Label` and `Milestone` sections.

I also changed the section name `Platform` for `Environment`. If you think that `Platform` or any other word like `Toolchain` for example fits better, let me know.

I also added a note at the bottom to invite users to ask questions in discourse. If you think it requires rewording, let me know. 

If you think the style of the comments should be changed (e.g. italics instead of or in addition to square brackets), let me know.

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+### Environment
+
+[Which your OS and compiler are]
+
+### Additional Information
+
+[Any additional information, configuration or data that might be necessary to reproduce the issue.]
+
+
```
Comment (thewtex) [inline ISSUE_TEMPLATE.md]: isses -> issues

Comment (jhlegarreta) [reply ISSUE_TEMPLATE.md]: Fixed.

**Comment by jhlegarreta** [comment]
@thewtex Thanks !

I've submitted a topic to gerrit:
http://review.source.kitware.com/#/c/22869/

to further review/merge this. I'm closing this PR then.

**Pull request closed without merging**